services/distribution/src/main/resources/main-config/v2/plausible-deniability-parameters.yaml

```diff
@@ -0,0 +1,5 @@
+#plausible-deniability-parameters:
+  # validation rules for checkInSizesInBytes:
+  #   * zero entries are allowed
+  #   * each entry must be >= 0
+  check-in-sizes-in-bytes: [10, 20]
```

Comment:
are we using those values anywhere for check in validation?

Reply:
Yes, we will use these values to fill up the protobuf for the app config. I used separate file for this one because it is an array of integer and the mapping from yaml to protobuf does not work as expected so I had to deserialise the yaml file.